src/common/Header.jsx

```diff
@@ -0,0 +1,51 @@
+import img1 from "../assets/images/logo.png";
+export default function Header() {
+  return (
+    <header className="ha-header-gtr">
+      <div className="ha-logo">
+        <a href="/index.html">
+          <img src={img1} alt="" />
```

Comment:
- Remove this **href** because HTML pages, don't exist or you can give "#" inside yo r **href**.
- Always give some value to the **alt** attribute so it helps for the screen reader to recognize a image